runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -471,6 +471,8 @@ private void hop(boolean previous)
 		currentWorldTypes.remove(WorldType.SKILL_TOTAL);
 		// Allow hopping from a high risk world to a non-high risk world
 		currentWorldTypes.remove(WorldType.PVP_HIGH_RISK);
```

Comment:
What is this? Not related to this PR but what? why is it in config.quickhopOutOfDanger too?

Reply:
Seems like that this line's a mistake and should be removed. But, I will make an assumption that `config.quickhopOutOfDanger()` is a setting someone can enable to be able to quick-swap to these risky worlds